app/views/components/heading.scala.html

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  *@
 
- @(headingMsg: String, headingSize: String = "heading-xlarge", headingClass: Option[String] = None)
+ @(headingMsg: String, headingSize: String = "heading-xlarge", headingClass: Option[String] = None, secondaryHeading: Option[String] = None)
```

Comment:
Also be aware that a similar change has been made in DIT-2271 so we'll need to resolve the conflict depending on who gets in first 😆 

https://github.com/hmrc/binding-tariff-trader-frontend/compare/DIT-2271#diff-8ec9426212e0d86f4a25add863abd441R21